src/server/models.ts

```diff
@@ -42,8 +44,9 @@ export const initDb = async (): Promise<Models> => {
 		Hackers: db.collection<HackerDbObject>('Hackers'),
 		Logins: db.collection<LoginDbObject>('logins'),
 		Mentors: db.collection<MentorDbObject>('mentors'),
-		Organizers: db.collection<OrganizerDbObject>('organizers'),
+		Organizers: db.collection<OrganizerDbObject>('Organizers'),
```

Comment:
Why rename?

Reply:
Sorry cause I set the collection name in db as Organizers and don't want to recreate it 🤦‍♀️

Reply:
Since this change was made, it opens up the question for why `Hackers` is uppercase. @cktang88 for confirmation the preferred way of naming in mongo is camelCase, right?

Reply:
Yes I believe camelCase is the convention